intellst-api/src/Entity/User.php

```diff
@@ -107,4 +114,16 @@ public function setEnterprise(?Enterprise $enterprise): self
 
         return $this;
     }
+    public function getUsername()
+    {
+        // TODO: Implement getUsername() method.
+    }
+    public function getSalt()
+    {
+        return null;
+    }
+    public function getRoles() :array
```

Comment:
spacing `(): array`
please, please, please, configure PSR-12, it's just few clicks

Reply:
stange i have aded PSR-12 to php storm, i don't understund